### PR TITLE
bottom modal

### PR DESCRIPTION
**Pull request by IbrohimIsroilov**

### CODE REVIEW

File: src/components/bottom_modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import {View, Text, StyleSheet, Dimensions} from 'react-native';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+  withTiming,
+} from 'react-native-reanimated';
+import {Gesture, GestureDetector} from 'react-native-gesture-handler';
+import {convertHeight, LightBlue2} from '../../utils';
+const {height: SCREEN_HIGHT} = Dimensions.get('window');
```
Comment (huytd2510) [inline src/components/bottom_modal/index.tsx:11]: I have const value is "deviceHeight". You can export it and use it

File: src/components/bottom_modal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    });
+
+  const rBottomSheetStyle = useAnimatedStyle(() => {
+    return {
+      transform: [{translateY: translateY.value}],
+    };
+  });
+
+  useEffect(() => {
+    scrollTo(-SCREEN_HIGHT / 50);
```
Comment (huytd2510) [inline src/components/bottom_modal/index.tsx:46]: should we need use useEffect in here?

File: src/components/bottom_modal/index.tsx (hunk 1 above)
Comment (IbrohimIsroilov) [reply src/components/bottom_modal/index.tsx:11]: ah ok got it

File: src/components/bottom_modal/index.tsx (hunk 2 above)
Comment (IbrohimIsroilov) [reply src/components/bottom_modal/index.tsx:46]: Actually, it is my first time using this library react-native-reanimated. That's why in some cases I think I need your help to finish the modal

**Pull request merged**